fuse.js

```diff
@@ -11,7 +11,11 @@ storybookManager.bundle("> index.js")
 const fuse = fsbx.FuseBox.init({
   homeDir: "src",
   tsConfig: "./tsconfig.json",
-  outFile: "./assets/reaction-force.js",
+  outFile: "./reaction-force.js",
+    sourceMap: {
+      bundleReference: "./reaction-force.js.map",
+      outFile: "./reaction-force.js.map",
+    },
```

Comment:
Yeah should probably have to change that at some point. I’m not even sure `assets` is the right place, but probably not in the root.